src/com/javarush/task/task04/task0409/Solution.java

```diff
@@ -0,0 +1,46 @@
+package com.javarush.task.task04.task0409;
+
+/*
+Ближайшее к 10
+*/
+
+public class Solution {
+    public static void main(String[] args) {
+        displayClosestToTen(8, 11);
+        displayClosestToTen(7, 14);
+    }
+
+    public static void displayClosestToTen(int a, int b) {
+        // напишите тут ваш код
+        System.out.printf("Из чисел %s и %s ", a, b);
+        if (abs(a - 10) >= abs(b - 10)) {
+
+            System.out.printf("число %s ближе к десяти\n" ,b);
+        } else {
+            System.out.printf("число %s ближе к десяти\n", a);
+        }
```

Comment:
Решение неплохое, но мне не нравится то что ты дважды вызываешь printf.
Вот альтернативный вариант (не руководство к действию).
```suggestion
        int closest;
        if (abs(a - 10) >= abs(b - 10)) {
            closest = b;
        } else {
            closest = a;
        }
        System.out.printf("число %s ближе к десяти\n" ,closest);
```

Reply:
блин, классно, что-то и не подумал